### PR TITLE
BUILD-6103 Create a standalone script to run million collections test

### PR DESCRIPTION
**Pull request by lukech**
Create a standalone script to run million collections test: 
- play all tasks Jenkins is taking care of right now e.g. clone a few git repos, merge wiredtiger develop contents, build mongo, start mongod, trigger the python script to run million collections testing
- make the script re-entrant e.g. move the test directory if exist, kill mongod process if any is running
- support one command argument to specify the directory in which test repos should be created

### CODE REVIEW

**Comment by lukech** [comment]
@agorrod the change is not ready for review yet - still need to run a few tests.

File: largescale/run-million-collection-test.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	    --exclude '*/tools/wtperf_stats' \
+	    --exclude '*/tools/wtstats/template' \
+	    -C wiredtiger -f ${tarball}
+	)
+}
+
+function build_mongo() { 
+	# Modify the build scripts
+	sed -i.bak 's/ggdb/gdwarf-2/' ./SConstruct
+	sed -i 's/stdint\.h>/stdint.h>\n        #include <stdbool.h>/g' src/third_party/wiredtiger/SConscript
```
Comment (agorrod) [inline largescale/run-million-collection-test.sh]: Are these still necessary?

Comment (lukech) [reply largescale/run-million-collection-test.sh]: These lines are copy-pasted from the Jenkins job. Will give it a try by removing the SConscript customisations.

Comment (lukech) [reply largescale/run-million-collection-test.sh]: Commented out the 2 `sed` lines and ran the test again on `nimer`, mongod was built successfully and the testing is running normally. It looks not necessary at more, will go ahead remove them in this script, and the Jenkins job.

Comment (agorrod) [reply largescale/run-million-collection-test.sh]: Thanks Luke

**Pull request merged**